install_config/web_console_customization.adoc

```diff
@@ -663,58 +663,55 @@ this customization in future upgrades if you modify existing category items.
 +
 [source, javascript]
 ----
-// Add Go to the Languages category
-var category = _.find(window.OPENSHIFT_CONSTANTS.CATALOG_CATEGORIES,
-{ id: 'languages' });
-category.items.splice(2,0,{ // Insert at the third spot
-  // Required.  Must be unique
+// Find the Languages category.
+var category = _.find(window.OPENSHIFT_CONSTANTS.SERVICE_CATALOG_CATEGORIES,
+                      { id: 'languages' });
+// Add Go as a new subcategory under Languages.
+category.subCategories.splice(2,0,{ // Insert at the third spot.
+  // Required. Must be unique.
   id: "go",
-  // Required
+  // Required.
   label: "Go",
-  // Optional.  If specified, defines a unique icon for this item
-  iconClass: "font-icon icon-go-gopher",
-  // Optional.  If specified, enables matching other tag values to this category
-  // item
-  categoryAliases: [
+  // Optional. If specified, defines a unique icon for this item.
+  icon: "icon-go-gopher",
+  // Required. Items with matching any tag will appear in this subcategory.
```

Comment:
this sentence doesnt read right to me, probably should have been "Items matching any"

same sentence appears again a couple times below